package.json

```diff
@@ -38,6 +38,7 @@
     "axios": "^0.27.2",
     "core-js": "^3.25.2",
     "http-status-codes": "^2.2.0",
+    "js-cookie": "^3.0.1",
```

Comment:
There's also `vue-cookies` but this one is more cross-platform and still super-easy to use.